File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Drawing/System.Drawing.Color/System.Drawing.ColorTranslator/Color.ToWin32.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library

Imports System.Drawing

Public Module Extensions_748

	''' <summary>
	'''     Translates the specified  structure to a Windows color.
	''' </summary>
	''' <param name="c">The  structure to translate.</param>
	''' <returns>The Windows color value.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function ToWin32(c As Color) As Int32
		Return ColorTranslator.ToWin32(c)
	End Function
End Module

File: src/Z.Drawing/System.Drawing.Color/System.Drawing.ColorTranslator/Color.ToWin32.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;
#if !NETSTANDARD
using System.Drawing;
#endif

public static partial class Extensions
{
#if !NETSTANDARD
    /// <summary>
    ///     Translates the specified  structure to a Windows color.
    /// </summary>
    /// <param name="c">The  structure to translate.</param>
    /// <returns>The Windows color value.</returns>
    public static Int32 ToWin32(this Color c)
    {
        return ColorTranslator.ToWin32(c);
    }
#endif
}